src/main/webapp/app/exercises/programming/manage/programming-exercise.component.html

```diff
@@ -192,20 +174,41 @@ <h4 class="mb-1 me-auto">
                                         <span class="d-md-inline" jhiTranslate="artemisApp.exercise.participations">Participations</span>
                                     </div>
                                 </a>
+                                <a
+                                    *ngIf="programmingExercise.isAtLeastTutor"
+                                    [routerLink]="['/course-management', courseId, 'programming-exercises', programmingExercise.id, 'scores']"
+                                    class="btn btn-info btn-sm me-1 mb-1"
+                                    style="display: flex; justify-content: center; align-items: center"
+                                >
```

Comment:
```suggestion
                                <a
                                    *ngIf="programmingExercise.isAtLeastTutor"
                                    [routerLink]="['/course-management', courseId, 'programming-exercises', programmingExercise.id, 'scores']"
                                    class="btn btn-info btn-sm me-1 mb-1"
                                >
```

The inline style is not needed for centering.